packages-next/admin-ui/src/components/__tests__/Pagination.test.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * @jest-environment jsdom
```

Comment:
We aren't able to augment the testEnvironment per package yet, so unfortunately this will have to happen per file till we have a solution for this